values/wire-server/prod-values.example.yaml

```diff
@@ -1,6 +1,7 @@
 # CHANGEME-PROD: All values here should be changed/reviewed
 tags:
   proxy: false # enable if you want/need giphy/youtube/etc proxying
+  legalhold: true
```

Comment:
suggestion: default to false, since legalhold is considered evil.

Reply:
Will do. This was set to true so it was included in the offline-ci but I'll make a custom `values.yaml` for that